build/Makefile

```diff
@@ -196,6 +204,12 @@ ifeq ($(ULINUX),1)
 endif
 
 #--------------------------------------------------------------------------------
+# Build AUOMS project
+
+$(AUOMS_TARGET_DIR)/auoms-bundle-test.sh : 
```

Comment:
I see, I interpreted it as a bundle for installing auoms for testing or mocking. I think having comments would really help. are other oss test scripts have the word "bundle" in the name?